Sources/Plot/API/HTMLComponents.swift

```diff
@@ -579,6 +579,96 @@ extension List: ComponentContainer where Items == ComponentGroup {
     }
 }
 
+// Component used to render an `<picture>` element for responsive image support.
+public struct Picture: Component {
+    /// Type used to define an  Picture source, which points to an image file (or set of them) and associated media queries (and other paramaters).
+    public struct Source {
+        /// image or images  to use - can have parameters
```

Comment:
Capitalize this documentation sentence. Also applies to the documentation for `sizes`. There's also a double space between "images" and "to use".